src/components/Register/Register.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+import { BASE_URL } from '../../helper/url'
+
+Register.propTypes = {
+  setToken: PropTypes.func,
+}
+
+export default function Register({ setToken }) {
```

Comment:
Same here as in your `Login` component! 
```suggestion
export default function Register({ onRegister }) {
```

Reply:
This component is almost the identical to your `Login` component, maybe think about just having one component?